modules/spring-config/src/main/java/org/mule/runtime/config/spring/dsl/spring/ObjectFactoryCommonConfigurator.java

```diff
@@ -6,17 +6,17 @@
  */
 package org.mule.runtime.config.spring.dsl.spring;
 
-import org.mule.runtime.config.spring.dsl.api.ComponentBuildingDefinition;
+import org.mule.runtime.dsl.api.component.AttributeDefinition;
 
 import java.util.Map;
 
 /**
- * Implementations of this interface are used configuration common attribute across a set of {@link ComponentBuildingDefinition}s.
+ * Implementations of this interface are used configuration common attribute across a set of {@link org.mule.runtime.dsl.api.component.ComponentBuildingDefinition}s.
```

Comment:
"Implementations of this interface are used configuration common attribute across a set of" is not clear, can you fix it?